hpcbench/toolbox/environment_modules.py

```diff
@@ -0,0 +1,16 @@
+import os
+import subprocess
+
+from . process import find_executable
+
+
+MODULECMD = find_executable('modulecmd', required=False)
+
+
+def module(*args):
```

Comment:
Maybe providing an interface to the purge, load, unload commands would be better. Why not just have a Module class. You can still provide a generic "command" method to catch all the rest?